src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts

```diff
@@ -290,21 +290,21 @@ export function getIndicatorsLabelAriaLabel(element: SettingsTreeSettingElement,
 	// Add sync ignored text
 	const ignoredSettings = getIgnoredSettings(getDefaultIgnoredSettings(), configurationService);
 	if (ignoredSettings.includes(element.setting.key)) {
-		ariaLabelSections.push(localize('syncIgnoredTitle', "This setting is ignored during sync"));
+		ariaLabelSections.push(localize('syncIgnoredAriaLabel', "Setting ignored during sync"));
 	}
 
 	// Add default override indicator text
 	const sourceToDisplay = getDefaultValueSourceToDisplay(element);
 	if (sourceToDisplay !== undefined) {
-		ariaLabelSections.push(localize('defaultOverriddenDetails', "Default setting value overridden by {0}", sourceToDisplay));
+		ariaLabelSections.push(localize('defaultOverriddenDetailsAriaLabel', "{0} overrides the default value", sourceToDisplay));
 	}
 
 	// Add text about default values being overridden in other languages
-	const otherLanguageOverridesStart = localize('defaultOverriddenListPreface', "The default value of the setting has also been overridden for the following languages:");
 	const otherLanguageOverridesList = element.overriddenDefaultsLanguageList
 		.map(language => languageService.getLanguageName(language)).join(', ');
 	if (element.overriddenDefaultsLanguageList.length) {
-		ariaLabelSections.push(`${otherLanguageOverridesStart} ${otherLanguageOverridesList}`);
+		const otherLanguageOverridesText = localize('defaultOverriddenLanguagesList', "Language-specific default values exist for {0}", otherLanguageOverridesList);
```

Comment:
Do we have a pluralization module? Or, does `{0} contribute(s) language-specific default values` with the `(s)` work?

Reply:
Not that I know of. The way I've done it in the past is to have two strings:
```
{0} contribute language-specific default values
{0} contributes language-specific default values
```

Reply:
Yup. No helper method that I am aware of.

Reply:
Will switch to using two strings during debt week.